app/code/Magento/Sales/Model/Order/Email/SenderBuilder.php

```diff
@@ -78,13 +78,14 @@ public function send()
     /**
      * Prepare and send copy email message
      *
+     * @param bool|false $toBcc
      * @return void
      */
-    public function sendCopyTo()
+    public function sendCopyTo($toBcc = false)
```

Comment:
Unfortunately, we are not allowed to change public methods signature according to the [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/).

Reply:
@dmytro-ch thanks for your review. But I assume if the class is not annotated by `@api`, we can update the signature. This class is kind of an utility class used by other two classes in the same folder, which are annotated by `@api`.

Reply:
@dmytro-ch I do think it's not the best idea to add this flag here too, but this the minimal change to fix this issue. But let me try with a better approach anyway.

Reply:
I just moved the check of `CopyMethod` to the caller class, so that to avoid introducing the signature. It looks more clear this way.